docs/tutorials/otio-serialized-schema.md

```diff
@@ -361,21 +361,65 @@ parameters:
 *documentation*:
 
 ```
-None
+Refers to a sequence of image files to be played back
+                    at a constant rate.
+
+                    The target urls for the images are determined by a concatentation of
+                    ``target_url_base``, ``name_prefix``, frame number (applying
+                    ``frame_zero_padding``) and ``name_suffix``.
+                    For instance, if the following values are used::
+                        target_url_base: "file:///show/sequence/shot/"
+                        name_prefix: "sample_image_sequence."
+                        name_suffix: ".exr"
+                        frame_zero_padding: 4
+
+                    Then calling ``ref.target_url_for_image_number(0)`` to fetch
+                    the first image in the sequence would retun::
+                        file:///show/sequence/shot/sample_image_sequence.0001.exr
+
+                    And calling ``ref.target_url_for_image_number(6)`` to fetch
+                    the 7th image in the sequence would return::
+                        file:///show/sequence/shot/sample_image_sequence.0007.exr
+
+                    If no zero padding should be included in frame numbers, set
+                    ``frame_zero_padding`` to either 0 or 1.
+
+                    In the context of the ImageSequenceReference, frame number
+                    refers to the number in the target url, image number refers
+                    to the frames in a zero-based index.
+                    This means that reguardless of the first frame number,
+                    ``ref.target_url_for_image_number(0)`` will always give the
+                    target url for that frame.
+
+                    The call below generates an ordered list of all the image
+                    urls::
+                        [ref.target_url_for_image_number(i) for i in range(ref.number_of_images_in_sequence)]
+
+                    The ``start_frame`` attribute sets what the frame number in
+                    the url will be for the first image in the sequence. For
+                    instance, if ``start_frame`` is set to ``5``, then a call
+                    ``ref.target_url_for_image_number(0)`` would return
+                    something like::
+                        file:///show/sequence/shot/sample_image_sequence.0005.exr
+
+                    Frame step sets number of frames to "skip". For instance, if
+                    ``start_frame`` is 1 and ``frame_step`` is 2, then the frame
+                    number sequence will be 1, 3, 5, 7, etc.
+                    
```

Comment:
This comment is great.  The only thing I would add to it is a (admittedly pedantic) note about how the timing is implemented.  IE played back at constant rate, but starting with the first frame in the sequence, which is on the screen for the time range [0, 1/rate), then the next image is shown for [1/rate, 2/rate) etc.  This maps image files on disk to time ranges in the media time.

Reply:
Also any limitations on what the rate can be set to would be good to note here